packages/card/README.md

```diff
@@ -10,14 +10,15 @@ npm install @crave/farmblocks-card
 
 ## API
 
-| Property | Description                                     | Type                                    |
-| -------- | ----------------------------------------------- | --------------------------------------- |
-| floating | if present, the card drops a bigger shadow      | boolean                                 |
-| type     | the type of card (reflects on background color) | string (one of "FEATURED" or "NEUTRAL") |
-| width    | a css value to be used as width                 | string                                  |
-| padding  | a css value to be used as padding               | string                                  |
-| overflow | a css value to be used as overflow              | string                                  |
-| children | the contents of the card                        | React.Node                              |
+| Property  | Description                                     | Type                                    |
+| --------- | ----------------------------------------------- | --------------------------------------- |
+| floating  | if present, the card drops a bigger shadow      | boolean                                 |
+| type      | the type of card (reflects on background color) | string (one of "FEATURED" or "NEUTRAL") |
+| width     | a css value to be used as width                 | string                                  |
+| padding   | a css value to be used as padding               | string                                  |
+| overflow  | a css value to be used as overflow              | string                                  |
+| children  | the contents of the card                        | React.Node                              |
+| boxShadow | specifies a custom border shadow                | string                                  |
```

Comment:
Minor: add an information that this prop overrides `floating`

Reply:
done (: